### PR TITLE
doc: fixup security release announcement

### PR DESCRIPTION
**Pull request by mhdawson**
- Fixup date show is show in news properly
- Add severity of second vulnerability

Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Review by richardlau — APPROVED** [review]

File: locale/en/blog/vulnerability/january-2021-security-releases.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-date: 2020-01-04T19:00:00.000Z
+date: 2021-01-04T19:30:00.000Z
```
Comment (APACGAMONDEBANK) [inline locale/en/blog/vulnerability/january-2021-security-releases.md:2]: APACGAMONDE send you greetings

File: locale/en/blog/vulnerability/january-2021-security-releases.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 category: vulnerability
 title: January 2021 Security Releases
 slug: january-2021-security-releases
```
Comment (APACGAMONDEBANK) [inline locale/en/blog/vulnerability/january-2021-security-releases.md:5]: Great thanks!

**Review by APACGAMONDEBANK — APPROVED** [review]

**Review by nschonni — APPROVED** [review]

File: locale/en/blog/vulnerability/january-2021-security-releases.md (hunk 1 above)
Comment (APACGAMONDEBANK) [inline locale/en/blog/vulnerability/january-2021-security-releases.md:2]: Thank you!

File: locale/en/blog/vulnerability/january-2021-security-releases.md (hunk 2 above)
Comment (APACGAMONDEBANK) [inline locale/en/blog/vulnerability/january-2021-security-releases.md:5]: Alright.; changes are approved.

File: locale/en/blog/vulnerability/january-2021-security-releases.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Impacts:
 
 Thank you to Felix Wilhelm from Google Project Zero for reporting this vulnerability.
 
-### HTTP Request Smuggling in nodejs (CVE-2020-8287)
+### HTTP Request Smuggling in nodejs (Low) (CVE-2020-8287)
```
Comment (APACGAMONDEBANK) [inline locale/en/blog/vulnerability/january-2021-security-releases.md:29]: OK; Thank you!

**Review by APACGAMONDEBANK — COMMENTED** [review]
Fix changes made from release announcement

**Comment by kasicka** [comment]
Should severity of the OpenSSL CVE also be mentioned in the post?

**Pull request merged**